tutorials/auto_scheduler/tune_network_cuda.py

```diff
@@ -186,7 +189,9 @@ def run_tuning():
     print("Begin tuning...")
     measure_ctx = auto_scheduler.LocalRPCMeasureContext(repeat=1, min_repeat_ms=400, timeout=10)
 
-    tuner = auto_scheduler.TaskScheduler(tasks, task_weights)
+    tuner = auto_scheduler.TaskScheduler(
+        tasks, task_weights, callbacks=[auto_scheduler.task_scheduler.PrintTableInfoCallback()]
```

Comment:
Can we make this the default callback to simplify the  API?

Reply:
If we make this as the default, then it means users have no way to turn it off. That's why I didn't do that, but I'm also fine if you prefer such semantic.

Reply:
How about setting `PrintTableInfoCallback` be default callback but allow users pass `callbacks=None` to turn  off ? We could add comment here to explain our design

Reply:
Sounds good, so when callbacks is not None, we override the default one with whatever users pass.

Reply:
Yes.

Reply:
Done